buildpacks/jvm-function-invoker/src/build.rs

```diff
@@ -0,0 +1,44 @@
+use std::path::PathBuf;
+
+use libcnb::data::launch::{Launch, Process};
+use libcnb::layer_lifecycle::execute_layer_lifecycle;
+use libcnb::{BuildContext, GenericPlatform};
+
+use crate::error::JvmFunctionInvokerBuildpackError;
+use crate::layers::bundle::BundleLayerLifecycle;
+use crate::layers::opt::OptLayerLifecycle;
+use crate::layers::runtime::RuntimeLayerLifecycle;
+use crate::JvmFunctionInvokerBuildpackMetadata;
+
+pub fn build(
+    context: BuildContext<GenericPlatform, JvmFunctionInvokerBuildpackMetadata>,
+) -> Result<(), libcnb::Error<JvmFunctionInvokerBuildpackError>> {
+    let run_sh_path: PathBuf = execute_layer_lifecycle("opt", OptLayerLifecycle {}, &context)?;
```

Comment:
I don't know what this is `execute_layer_lifecycle`. Docs are https://github.com/Malax/libcnb.rs/blob/6a94d587814bce9f2a2f411cd655c2c09063a033/src/layer_lifecycle.rs#L154-L155 points to https://github.com/Malax/libcnb.rs/blob/6a94d587814bce9f2a2f411cd655c2c09063a033/src/layer_lifecycle.rs#L15-L17. No examples, not terribly specific.

It looks like LayerLifecycle responds to:
- create
- validate
- update
- etc.

Based on the usage I'm seeing that it, well, executes that layer. This seems to be the primary entry point for how the buildpack is composed.